installation/README.md

```diff
@@ -0,0 +1,161 @@
+# Production deployment using docker
+
+### Setup Letsencrypt Nginx Proxy Companion
+
+DNS needs to be configured for following to work
+
+Read more: https://github.com/evertramos/docker-compose-letsencrypt-nginx-proxy-companion
+
+```sh
+cd $HOME
+git clone https://github.com/evertramos/docker-compose-letsencrypt-nginx-proxy-companion.git
+cd docker-compose-letsencrypt-nginx-proxy-companion
+cp .env.sample .env
+./start.sh
+```
+
+### Clone frappe_docker repository
+
+```sh
+cd $HOME
+git clone https://github.com/frappe/frappe_docker.git
+cd frappe_docker
+cp installation/env-example .env
```

Comment:
`.env` should be in project directory, so `installation/.env`.

Reply:
I thought it is suppose to be in directory where `docker-compose` is being executed?

Reply:
I though the same, but then I tested it.

Reply:
@MarekPikula

https://github.com/frappe/frappe_docker/wiki/Troubleshoot#docker-compose-does-not-recognize-variables-from-env-file

Reply:
Interesting, thanks :+1: